CanvasApi.Client/EnrollmentTerms/Models/EnrollmentTermRetrieveOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using CanvasApi.Client.EnrollmentTerms.Enums;
+
+namespace CanvasApi.Client.EnrollmentTerms.Models
+{
+    internal class EnrollmentTermRetrieveOptions : IEnrollmentTermRetrieveOptions
+    {
+        public IEnumerable<EnrollmentTermIncludes> Include { get; set; }
```

Comment:
Looks like the enum name is EnrollmentTermListIncludes. 

The build server is rejecting this file.